crates/volta-core/src/run/binary.rs

```diff
@@ -101,3 +107,55 @@ where
         },
     )
 }
+
+pub struct DefaultBinary {
```

Comment:
Can we add a doc-string to explain what this is for? I *think* I follow based on how it's used, but some lightweight docs would be 😍.

Reply:
Definitely, I'll write something up.